httpx/config.py

```diff
@@ -90,6 +91,12 @@ def load_ssl_context(
     ) -> ssl.SSLContext:
         http_versions = HTTPVersionConfig() if http_versions is None else http_versions
 
+        if self.trust_env:
```

Comment:
Can we move this section into the `load_ssl_context_no_verify()`? Since we're only doing the loading if we've got verify active anyways.

Also is there any harm in having `SSLConfig.verify` default to `True` in the constructor? It defaults to `True` on the client anyways.

Reply:
Did you mean `load_ssl_context_verify`? when `verify` active, `load_ssl_context_verify` is called.

Reply:
Yep! Big typo there haha

Reply:
:) i'm moving